class.jetpack-gutenberg.php

```diff
@@ -89,6 +102,52 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a block is able to run with current wp or gutenburg plugin versions
+	 *
+	 * @param string $core_wp_version The minimum wp core version needed to run the block.
+	 * @param string $plugin_version The minimum gutenberg plugin version needed to run the block.
+	 *
+	 * @since 8.4.0
+	 *
+	 * @return boolean True if the block is gated due to current wp or plugin versions.
+	 */
+	public static function is_block_version_gated( $core_wp_version, $plugin_version ) {
+		global $wp_version;
+
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
Is there a reason why WPCOM will always return `false`? It seems like there are reasonable situations where it may not have the necessary version of Core or Gutenberg (for example, I believe some VIPs are running older versions of Gutenberg).

Reply:
I was thinking that wpcom simple sites currently have the latest gutenberg release - but if there are VIP sites running older versions which have IS_WPCOM set then I will take a look and see how we can identify which gutenberg version is active for any site which has IS_WPCOM true.